app/services/jobs/return-logs/create-current-return-cycle.service.js

```diff
@@ -14,16 +14,44 @@ const {
 const ReturnCycleModel = require('../../../models/return-cycle.model.js')
 
 /**
- * Given a return cycle type, it generates the cycle data based on the current date and creates it
- *
- * @param {boolean} summer - true if creating the current summer return cycle, else false
+ * Verify that the town most recent return cycles are the most current ones. If they're not create the missing cycles.
```

Comment:
The convention is that the summary for the function is the same as the module description.

I believe this will be a better summary, though, once the references to 'the town' are dealt with!

I wasn't sure where you were going with this, so I couldn't suggest anything! 😂